swift/aperture/Recorder.swift

```diff
@@ -79,22 +79,16 @@ final class Recorder: NSObject, AVCaptureFileOutputRecordingDelegate {
     session.stopRunning()
   }
 
-  internal func capture(_ captureOutput: AVCaptureFileOutput, didStartRecordingToOutputFileAt fileURL: URL, fromConnections connections: [Any]) {
-    print("R") // At this point the recording really started
+  func capture(_ captureOutput: AVCaptureFileOutput, didStartRecordingToOutputFileAt fileURL: URL, fromConnections connections: [Any]) {
+    onStart?()
   }
 
-  internal func capture(_ captureOutput: AVCaptureFileOutput, didFinishRecordingToOutputFileAt outputFileURL: URL, fromConnections connections: [Any], error: Error!) {
-    // TODO: Make `stop()` accept a callback that is called when this method is called and do the exiting in `main.swift`
-    if error != nil {
-      // Don't print useless "Stop Recording" error
-      if error._code != -11806 {
-        print(error)
-        exit(1)
-      } else {
-        exit(0)
-      }
+  func capture(_ captureOutput: AVCaptureFileOutput, didFinishRecordingToOutputFileAt outputFileURL: URL, fromConnections connections: [Any], error: Error) {
```

Comment:
You changed the method signature by removing the force casting on the error. This function will never get called. (Its no longer the same method as the delegate protocol..)